Cargo.toml

```diff
@@ -19,21 +19,21 @@ resolver = "2"
 [workspace.dependencies]
 clap = "4.4"
 sapling-crypto = "0.1.2"
-zcash_address = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_address-0.3.2-128-g11bcb838" } # git tag -s $(git describe --dirty)
-zcash_client_backend = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_address-0.3.2-128-g11bcb838", features = ["lightwalletd-tonic"] }
-zcash_encoding = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_address-0.3.2-128-g11bcb838" }
-zcash_keys = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zcash_address-0.3.2-128-g11bcb838", features = ["orchard"] }
+zcash_address = { git = "https://github.com/zingolabs/librustzcash.git", branch = "simplified_external_change_address" } # git tag -s $(git describe --dirty)
```

Comment:
As mentioned elsewhere I am -1 on this change.

Reply:
Well...   maybe this is OK for `dev`, but not `stable`?  I get the convenience for rapid development.   But how can we enforce that this doesn't get into stable?